inc/api.class.php

```diff
@@ -115,7 +115,7 @@ public function initApi() {
 
       // check if api is enabled
       if (!$CFG_GLPI['enable_api']) {
-         $this->returnError(__("API disabled"), "", "", false);
+         $this->returnError(__("API disabled"), 403, "", false);
```

Comment:
Not entirely sure bout the 403

Reply:
Me neither. I thought about putting 418 :). Or I can put 400, to keep same behaviour as current state.

Reply:
ping @btry ?

Reply:
418 ? no way ! GLPI is NOT a teapot :)

I Think that 403 is the best choice

Reply:
So, 403 or 400 ?

Reply:
Nevermind, message is explicit, this should be enough